x-pack/plugins/actions/server/application/connector/methods/get_all/get_all.ts

```diff
@@ -9,17 +9,17 @@
  * Get all actions with in-memory connectors
  */
 import * as estypes from '@elastic/elasticsearch/lib/api/typesWithBodyKey';
-import { connectorSchema } from '../../schemas';
+import { connectorWithExtraFindDataSchema } from '../../schemas';
```

Comment:
WDTY of `connectorWithRefCount` as a name?

Reply:
Same here, are we sure that we will not add more data to find response?